sdks/java/core/pom.xml

```diff
@@ -84,54 +84,6 @@
         <artifactId>maven-jar-plugin</artifactId>
       </plugin>
 
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-shade-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>bundle-and-repackage</id>
-            <phase>package</phase>
-            <goals>
-              <goal>shade</goal>
-            </goals>
-            <configuration>
-              <shadeTestJar>true</shadeTestJar>
-              <artifactSet>
-                <includes>
-                  <include>com.google.guava:guava</include>
-                </includes>
-              </artifactSet>
-              <filters>
-                <filter>
-                  <artifact>*:*</artifact>
-                  <excludes>
-                    <exclude>META-INF/*.SF</exclude>
-                    <exclude>META-INF/*.DSA</exclude>
-                    <exclude>META-INF/*.RSA</exclude>
-                  </excludes>
-                </filter>
-              </filters>
-              <relocations>
-                <!-- TODO: Once ready, change the following pattern to 'com' only, 
-                  exclude 'org.apache.beam.**', and remove the second relocation. -->
-                <relocation>
-                  <pattern>com.google.common</pattern>
-                  <excludes>
-                    <!-- com.google.common is too generic, need to exclude guava-testlib -->
-                    <exclude>com.google.common.**.testing.*</exclude>
```

Comment:
Thanks @swegner  for your comment!

All dependencies on `guava-testlib` in the project currently are in scope `test`.

I removed shading of test jars in modules which matched the shading rules that are in the root pom in my branch (They now do not shade their test jars)

The only test jar that remains with shading is the one of Dataflow runner, which still has this exclusion in its shade-plugin configuration: [google-cloud-dataflow-java/pom.xml](https://github.com/aviemzur/beam/blob/shade-guava-generically/runners/google-cloud-dataflow-java/pom.xml#L138)

Is this sufficient?